apps/quickbooks_online/utils.py

```
@@ -465,19 +464,23 @@ def sync_departments(self):
-            with transaction.atomic():
-                DestinationAttribute.objects.filter(
-                    attribute_type='DEPARTMENT',
-                    workspace_id=self.workspace_id,
-                    destination_id__in=batch,
-                    active=True
-                ).update(active=False, updated_at=timezone.now())
+        inactive_departments = self.connection.departments.get_inactive(sync_after)
 
-        # qbo_sync_timestamp.department_synced_at = timezone.now()
-        # qbo_sync_timestamp.save(update_fields=['department_synced_at', 'updated_at'])
+        for inactive_departments in inactive_departments:
```

Ruff (0.12.2)

Loop control variable `inactive_departments` overrides iterable it iterates

(B020)